ubuntu_image/classic_builder.py

```diff
@@ -48,6 +48,9 @@ def prepare_gadget_tree(self):
         super().prepare_gadget_tree()
 
     def prepare_image(self):
+        if self.args.filesystem:
+            return super().prepare_image()
```

Comment:
I know this probably seemed better by not making the diff too big, but I'd prefer if maybe we didn't do a duplicate super().prepare_image() call here. Maybe just move the conditional if not self.args.filesystem: to before the try - except clause, remove the else, add an empty return to the exception handler and then, at the end of the function, just do the super().prepare_image() call.